data/json/items/comestibles/med.json

```diff
@@ -15,16 +15,17 @@
         "color" : "red",
         "use_action" : {
             "type" : "heal",
-            "limb_power" : 14,
-            "head_power" : 10,
-            "torso_power" : 18,
+            "limb_power" : 0,
+            "head_power" : 0,
+            "torso_power" : 0,
```

Comment:
Both `head_power` and `torso_power` equals to zero by default and can be completely removed.